### PR TITLE
Proposed 1.9.2

### PR DESCRIPTION
**Pull request by manojsdoshi**

### CODE REVIEW

**Review by nbougalis — APPROVED** [review]

File: RELEASENOTES.md [hunk 1]
```diff
@@ -7,6 +7,69 @@ This document contains the release notes for `rippled`, the reference server imp
...
+- **Fix and refactor spinlock code.** The spinlock code, which protects the `SHAMapInnerNode` child lists, had a mistake that allowed the same child to be repeatedly locked under some circumstances. Fixed this bug and improved the spinlock code to make it easier to use correctly and easier to verify that the code works correctly. ([#4201](https://github.com/XRPLF/rippled/pull/4201))
+- **Improve comments and contributor documentation.** Various minor documentation changes including some to reflect the fact that the source code repository is now owned by the XRP Ledger Foundation. ([#4214](https://github.com/XRPLF/rippled/pull/4214), [#4179](https://github.com/XRPLF/rippled/pull/4179), [#4222](https://github.com/XRPLF/rippled/pull/4222))
+- **Introduces a new API book_changes to provide information in a format that is useful for building charts that highlight DEX activity at a per-ledger level.**  ([#4212](https://github.com/XRPLF/rippled/pull/4212))
+
+## Contributions
+
+### GitHub
+
+The public source code repository for `rippled` is hosted on GitHub at <https://github.com/ripple/rippled>.
```
Comment (ximinez) [inline RELEASENOTES.md:55]: Should this URL be https://github.com/XRPLF/rippled?

File: RELEASENOTES.md [hunk 2]
```diff
@@ -7,6 +7,69 @@ This document contains the release notes for `rippled`, the reference server imp
...
+
+- Chenna Keshava B S <ckbs.keshava56@gmail.com>
+- Ed Hennis <ed@ripple.com>
+- Ikko Ashimine <eltociear@gmail.com>
+- Nik Bougalis <nikb@bougalis.net>
+- Richard Holland <richard.holland@starstone.co.nz>
+- Scott Schurr <scott@ripple.com>
+- Scott Determan <scott.determan@yahoo.com>
+
+For a real-time view of all lifetime contributors, including links to the commits made by each, please visit the "Contributors" section of the GitHub repository: <https://github.com/ripple/rippled/graphs/contributors>.
```
Comment (ximinez) [inline RELEASENOTES.md:71]: Should this URL be https://github.com/XRPLF/rippled/graphs/contributors?

**Review by ximinez — CHANGES_REQUESTED** [review]

File: RELEASENOTES.md (hunk 1 above)
Comment (nbougalis) [reply RELEASENOTES.md:55]: It should be, thanks!

File: RELEASENOTES.md (hunk 2 above)
Comment (nbougalis) [reply RELEASENOTES.md:71]: It should be, thanks!

File: RELEASENOTES.md (hunk 1 above)
Comment (manojsdoshi) [reply RELEASENOTES.md]: Thanks @ximinez

File: RELEASENOTES.md (hunk 2 above)
Comment (manojsdoshi) [reply RELEASENOTES.md]: Thanks @ximinez Fixed

**Comment by nbougalis** [comment]
This was merged; please note that this was functionally identical to the previously merged **1.9.2-rc1** via #4219 and which received two approvals.

**Pull request merged**